frontend/public/components/modals/create-namespace-modal.jsx

```diff
@@ -108,108 +109,115 @@ const CreateNamespaceModal = connect(
     }
 
     render() {
-      const label = this.props.createProject ? 'Project' : 'Namespace';
       const defaultNetworkPolicies = {
```

Comment:
We should internationalize these as well.

Reply:
Replaced the `label` const since it was only being used in `<ModalTitle>` and translated each string  with
```<ModalTitle>
                {this.props.createProject
                  ? t('public~Create Project')
                  : t('public~Create Namespace')}
              </ModalTitle>
```
https://github.com/openshift/console/pull/8204/files#diff-658c9e962611ba32543853d54f224c95f31993dc0f574dd7687ece2b53514a86R125-R127